be/src/http/action/update_config_action.cpp

```diff
@@ -74,6 +74,16 @@ void UpdateConfigAction::handle(HttpRequest* req) {
             cache_limit = _exec_env->check_storage_page_cache_size(cache_limit);
             StoragePageCache::instance()->set_capacity(cache_limit);
         });
+        _config_callback.emplace("disable_storage_page_cache", [&]() {
+            LOG(INFO) << "set disable_storage_page_cache:" << config::disable_storage_page_cache;
```

Comment:
This log is not necessary. There is another log at the beginning of this function.

Reply:
> This log is not necessary. There is another log at the beginning of this function.

updated